### ISSUE TITLE
Webview page title

### ISSUE THREAD

**Issue by AdrianoPatrizio** [nwjs/nw.js#5514]
Chrome API for Webview tag, not implement a method for read the page title or title-change event.

**Comment by ghostoy** [comment]
We see to support it.

Currently I believe you can achieve this by using a combination of [contentLoad](https://developer.chrome.com/apps/tags/webview#event-contentload) event, [executeScript](https://developer.chrome.com/apps/tags/webview#method-executeScript) method and [DOMSubtreeModified](http://stackoverflow.com/questions/2497200/how-to-listen-for-changes-to-the-title-element) event.

**Comment by bambooCZ** [comment]
Pull request: https://github.com/nwjs/nw.js/pull/5754

**Comment by nwittwer** [comment]
> Currently I believe you can achieve this by using a combination of [contentLoad](https://developer.chrome.com/apps/tags/webview#event-contentload) event, [executeScript](https://developer.chrome.com/apps/tags/webview#method-executeScript) method and [DOMSubtreeModified](http://stackoverflow.com/questions/2497200/how-to-listen-for-changes-to-the-title-element) event.

Yep, getting the title of the `<webview>` is possible to achieve (as a workaround) with the `contentload` event and `executeScript()`, and in addition, using `postMessage()`. I just added a [StackOverflow answer](https://stackoverflow.com/questions/54155507/how-can-i-access-the-contents-inside-of-a-webview-nw-js-electron/54155508) specifically about this to hopefully help others in the future.

You can actually get access to the entire `<webview>`'s DOM this way. Which means you can easily return the `document.title` and anything else you could want. From there, you could theoretically also use—as @ghostoy mentioned—`DOMSubtreeModified`, to create your own title-change event.

**Comment by hexianzhi** [comment]
It works with @nwittwer StackOverflow answer.

**Comment by Cherish-xzw** [comment]
Try this
```js
      webview.executeScript({
        code: `document.title`
      }, result => {
        this.title = result[0];
      })
```
